AlphaWallet/Transfer/Controllers/TransactionConfigurator.swift

```diff
@@ -112,6 +112,23 @@ class TransactionConfigurator {
         }
     }
 
+    // Generic function to derive the typical acceptable gas price on each network
+    static public func estimateGasPrice(server: RPCServer, completion: @escaping (BigInt) -> Void) {
```

Comment:
@hboon I know you don't like statics but this function is super generic and you shouldn't need to initialise a class just to use it

Reply:
Nah. It's fine.